spec/macros/resource_macros.rb

```diff
@@ -78,6 +80,8 @@ def it_should_be_updatable(attribute, value = "TESTDATA", extra = {})
       end
 
       after(:all) do
+        return unless @object.id
+
```

Comment:
These two changes help error readability. The errors in "after test" hide the actual error, the one in creation.

If there's no id, let's not even try to delete the resource.